presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -989,6 +990,7 @@ private void printTableScanInfo(TableScanNode node, int indent)
                     predicate.getDomains().get()
                             .entrySet().stream()
                             .filter(entry -> !outputs.contains(entry.getKey()))
+                            .sorted(Comparator.comparing(o -> o.getKey().toString()))
```

Comment:
Can you please add a test to `com.facebook.presto.tests.TestQueryPlanDeterminism` that would fail without your change here?

Reply:
@kokosing Grzegorz, this functionality requires partitioned tables. TestQueryPlanDeterminism doesn't have any.

Reply:
Isn't TPCH "partitioned"? See `orders.orderstatus`, `part.container` and `part.type`

Reply:
@kokosing That's right. I didn't notice. I couldn't reproduce the issue using these tables though. In general, this has been tricky to reproduce. One repro we have involves a complicated query and a table with 7 partition keys.